frontend/www/js/omegaup/components/user/Profilev2.vue

```diff
@@ -0,0 +1,268 @@
+                ></omegaup-grid-paginator>
+                <omegaup-grid-paginator
+                  v-bind:columns="3"
+                  v-bind:items="createdProblems"
+                  v-bind:items-per-page="30"
+                  v-bind:title="T.profileCreatedProblems"
+                  class="mb-3"
+                ></omegaup-grid-paginator>
+              </div>
+              <div
+                class="tab-pane fade show active"
+                role="tab"
+                aria-labelledby="nav-contests-tab"
+                v-if="selectedTab == 'contests'"
+              >
+                <omegaup-grid-paginator
+                  v-bind:columns="1"
+                  v-bind:items="contests"
+                  v-bind:items-per-page="15"
+                >
+                  <template slot="table-header">
+                    <thead>
+                      <tr>
+                        <th>{{ T.profileContestsTableContest }}</th>
+                        <th class="numericColumn">
+                          {{ T.profileContestsTablePlace }}
+                        </th>
+                      </tr>
+                    </thead>
+                  </template>
+                </omegaup-grid-paginator>
+              </div>
+              <div
+                class="tab-pane fade show active"
+                role="tab"
+                aria-labelledby="nav-user-info-tab"
+                v-if="selectedTab == 'data'"
+              >
+                <omegaup-user-basicinfo
+                  v-bind:profile="profile"
+                  v-bind:rank="rank"
+                ></omegaup-user-basicinfo>
+              </div>
+              <div
+                class="tab-pane fade show active"
+                role="tab"
+                aria-labelledby="nav-charts-tab"
+                v-if="selectedTab == 'charts'"
+              >
+                <omegaup-user-charts
+                  v-bind:data="charts"
+                  v-bind:username="profile.username"
+                  v-bind:periodStatisticOptions="this.periodStatisticOptions"
+                  v-on:emit-update-period-statistics="
+                    (profileComponent, categories, data) =>
+                      $emit(
+                        'update-period-statistics',
+                        profileComponent,
+                        categories,
+                        data,
+                      )
+                  "
+                  v-bind:aggregateStatisticOptions="
+                    this.aggregateStatisticOptions
+                  "
+                  v-on:emit-update-aggregate-statistics="
+                    profileComponent =>
+                      $emit('update-aggregate-statistics', profileComponent)
+                  "
+                  v-if="charts"
+                ></omegaup-user-charts>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style>
```

Comment:
```suggestion
<style lang="scss" scoped>
```

sin el [`scoped`](https://vue-loader.vuejs.org/guide/scoped-css.html), L225 se va a escapar y afectaría otras cosas fuera de este componente!